packages/twenty-front/src/modules/object-record/utils/computeOptimisticCreateRecordBaseRecordInput.ts

```diff
@@ -0,0 +1,24 @@
+import { ObjectMetadataItem } from '@/object-metadata/types/ObjectMetadataItem';
+import { hasObjectMetadataItemFieldCreatedBy } from '@/object-metadata/utils/hasObjectMetadataItemFieldCreatedBy';
+import { hasObjectMetadataItemPositionField } from '@/object-metadata/utils/hasObjectMetadataItemPositionField';
+import { FieldActorForInputValue } from '@/object-record/record-field/types/FieldMetadata';
+import { ObjectRecord } from '@/object-record/types/ObjectRecord';
+
+export const computeOptimisticCreateRecordBaseRecordInput = (
+  objectMetadataItem: ObjectMetadataItem,
+) => {
+  const baseRecordInput: Partial<ObjectRecord> = {};
+
+  if (hasObjectMetadataItemFieldCreatedBy(objectMetadataItem)) {
+    baseRecordInput.createdBy = {
+      source: 'MANUAL',
+      context: {},
+    } satisfies FieldActorForInputValue;
+  }
+
+  if (hasObjectMetadataItemPositionField(objectMetadataItem)) {
+    baseRecordInput.position = Number.NEGATIVE_INFINITY;
```

Comment:
Should we really use this NEGATIVE_INFINITY ?

Reply:
Indeed this might seems a bit radical but calculating the next index would quite complex for a very small plus value regarding this solution here

Reply:
Note that this is note sent to the api and will override on api response